unique_toolkit/unique_toolkit/chat/service.py

```diff
@@ -0,0 +1,182 @@
+import unique_sdk
+from unique_sdk.utils.chat_history import load_history
+
+from unique_toolkit.performance.async_wrapper import to_async
+
+from .state import ChatState
+
+
+class ChatService:
```

Comment:
Can you please include all my implementations for now, I will make a separate commit for them.